src/worldstate/include/routines/Gate/PassGate.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Created By: Joel Ahn
+ * Created On: March 5th, 2018
+ * Description: World State FSM node. Checks to see if
+ *              robot still needs to pass gate.
+ */
+
+#ifndef PROJECT_PASSGATE_H
+#define PROJECT_PASSGATE_H
+
+#include "State.h"
+#include <constants.h>
+#include <gate_detect/gateDetectMsg.h>
+
+/*** Communicating Class {alignWithGate, locatingGate, passGate} ***/
```

Comment:
Not really sure what we're trying to say here, we should either remove or make this more clear.

Reply:
Removed!